msprime/simulations.py

```diff
@@ -1365,6 +1368,139 @@ def print_history(self, output=sys.stdout):
             self._print_populations(epoch, output)
             print(file=output)
 
+    def population_size_trajectory(self, end, num_steps=10):
+        """
+        This function computes the defined effective population size
```

Comment:
To disambiguate this from the effective population size that is 1/(coal rate), how about:
```
        This function returns an array of per-population effective population sizes,
        as defined by the demographic model. These are just the `population_size`
        parameters of the model, modified by any population growth rates.
        The sizes are computed at a regular grid of time steps, between time 0 and `end`
        units of time ago.

        Returns a tuple where the first value
        is an array giving the times at which population sizes have been computed,
        the second is a num_steps-by-num_pops ndarray containing the effective
        population sizes for each subpopulation individually, at the corresponding step.
```